### PR TITLE
Upgrade request validation module to v1.3.22

### PR DESCRIPTION
**Pull request by selfxp**
Use the latest version of the `api-gateway-request-validation` module

### CODE REVIEW

**Review by buskamuza — APPROVED** [review]

File: Dockerfile-debian [hunk 1]
```diff
@@ -244,30 +213,76 @@ RUN echo " ... installing api-gateway-request-tracking ..." \
...
+    && rm -rf /tmp/api-gateway
+
 
-RUN apt-get -y remove --purge --allow-remove-essential make unzip gcc git zlib1g-dev build-essential \
+RUN apt-get -y remove --purge --allow-remove-essential make unzip gcc git zlib1g-dev build-essential libpcre3-dev \
     && apt-get -y autoremove
 
+
+# Final image stage
+FROM --platform=${TARGETPLATFORM:-linux/amd64} debian:10-slim
```
Comment (YevSent) [inline Dockerfile-debian:256]: Is there any reason why we can't use Debian 12 here? The Debian 10 is EOL June 2024

File: Dockerfile-debian [hunk 2]
```diff
@@ -66,27 +89,25 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update \
         git \
         zlib1g-dev \
         gcc \
-        python2.7 \
         iproute2 \
         htop
 
 RUN cd /tmp \
-    && curl -fSL https://www.openssl.org/source/openssl-${RESTY_OPENSSL_VERSION}.tar.gz -o openssl-${RESTY_OPENSSL_VERSION}.tar.gz \
+    && curl -fSLk https://www.openssl.org/source/openssl-${RESTY_OPENSSL_VERSION}.tar.gz -o openssl-${RESTY_OPENSSL_VERSION}.tar.gz \
```
Comment (buskamuza) [inline Dockerfile-debian:98]: Why do we need to use `-k` option everywhere?

File: Dockerfile-debian (hunk 1 above)
Comment (selfxp) [reply Dockerfile-debian:256]: There are some Adobe specific image build limitations, that don't allow us to upgrade to `bookworm` at the moment. However we're planning to change this by EOL of 10.

File: Dockerfile-debian (hunk 2 above)
Comment (selfxp) [reply Dockerfile-debian:98]: Download breaks otherwise

**Review by buskamuza — APPROVED** [review]

**Review by ac4922 — APPROVED** [review]
LGTM

**Pull request merged**